pkgs/applications/audio/ardour/default.nix

```diff
@@ -183,8 +177,8 @@ stdenv.mkDerivation rec {
       https://community.ardour.org/donate
     '';
     homepage = "https://ardour.org/";
-    license = licenses.gpl2Plus;
+    license = licenses.gpl2;
```

Comment:
Just for reference, I was originally asked to amend this [here](https://github.com/NixOS/nixpkgs/pull/115121#issuecomment-790912722) due to `gpl2` being deprecated.

Reply:
`gpl2Plus` seems to be incorrect indeed, though.

Reply:
Ahh thanks for checking!

Reply:
Ahh thanks for checking!

Reply:
I think [this line](https://github.com/Ardour/ardour/blob/master/libs/ardour/ardour/ardour.h#L11-L12) in the src is what lead me to think this was `gpl2Plus` - is there somewhere the ardour folks specify it's gpl2 only?

Reply:
I did not test the video stuff, since I never used it.

Reply:
There's https://community.ardour.org/download which says "version 2, or (at your option) any later version." So `gpl2Plus`

Reply:
Okay, got mislead by the site's footer then :)